sphinx/builders/latex/__init__.py

```diff
@@ -405,6 +405,16 @@ def default_latex_use_xindy(config):
     return config.latex_engine in {'xelatex', 'lualatex'}
 
 
+def default_latex_documents(config):
+    # type: (Config) -> List[Tuple[unicode, unicode, unicode, unicode, unicode]]
+    """ Better default latex_use_xindy settings. """
```

Comment:
is this docstring mentioning `latex_use_xindy` an unrelated left-over?

Reply:
oh, it's my wrong.